adapters/infytv/infytv.go

```diff
@@ -0,0 +1,89 @@
+package infytv
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/mxmCherry/openrtb/v15/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type Adapter struct {
```

Comment:
Actually, we would prefer the name `adapter`. The beginning upper case letter tells Go to export this type and there is no need for that. Sorry to contradict with @hhhjort here.

Reply:
@SyntaxNode Updated to `adapter`.